esque/cli/commands.py

```diff
@@ -14,16 +15,22 @@
 from esque.cli.output import bold, pretty, pretty_topic_diffs, get_output_new_topics, blue_bold, green_bold
 from esque.clients import FileConsumer, FileProducer, AvroFileProducer, AvroFileConsumer, PingConsumer, PingProducer
 from esque.cluster import Cluster
-from esque.config import PING_TOPIC, Config, PING_GROUP_ID
+from esque.config import PING_TOPIC, Config, PING_GROUP_ID, config_dir, sample_config_path, config_path
 from esque.consumergroup import ConsumerGroupController
 from esque.errors import ConsumerGroupDoesNotExistException, ContextNotDefinedException, TopicAlreadyExistsException
 from esque.topic import TopicController
 
 
-@click.group(help="(Kafka-)esque.")
+@click.group(help="esque - an operational kafka tool.", invoke_without_command=True)
+@click.option("--recreate-config", is_flag=True, default=False, help="Overwrites the config with the sample config.")
```

Comment:
i'd personally prefer to have this as command, not as an option for the base `esque` command

Reply:
I don’t. But that’s up for discussion